src/main/webapp/app/exam/manage/student-exams/student-exam-timeline/student-exam-timeline.component.scss

```diff
@@ -0,0 +1,49 @@
+::ng-deep {
```

Comment:
Please see our [client guideline](https://docs.artemis.cit.tum.de/dev/guidelines/client) for ng-deep usage. 
It is advised to use a ` :host` in front of the command.